samples/autoheal-config.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: autoheal.redhat.com/v1alpha1
+kind: AutohealConfig
+metadata:
+  name: autoheal-config
+spec:
+  syncPeriod: 1h
```

```suggestion
  syncPeriod: 1h
  secretName: autoheal-config-secret
```
Do we need to add the secretname to this sample CR?

---

oh yes ill add it